src/plugincode/post_scan.py

```diff
@@ -57,6 +73,9 @@ def initialize():
     NOTE: this defines the entry points for use in setup.py
     """
     post_scan_plugins.load_setuptools_entrypoints('scancode_post_scan')
+    for name, plugin in get_post_scan_plugins().items():
+        if not issubclass(plugin, PostScanPlugin):
+            raise Exception('Invalid post-scan plugin "%(name)s": does not extend "plugincode.post_scan.PostScanPlugin".' % locals())
```

Comment:
I guess this is OK for now, though eventually we may not care that a plugin extends a certain class, ass long as it quacks like one.